src/main/java/modal/StringAddCalculator.java

```diff
@@ -0,0 +1,59 @@
+package modal;
+
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static String DELIMITER = "[,:]";
+    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\\n(.*)";
+
+    public static int splitAndSum(String inputString) {
+        if(checkNullinput(inputString)) {
+            return 0;
+        }
+        String[] splitString = makeSplitString(inputString);
+        return sumNumbers(splitString);
+    }
+
+    private static int checkMinus(String stringNum) {
+        int number = 0;
+        try {
+            number = Integer.parseInt(stringNum);
+            if(number < 0) {
+                throw new RuntimeException();
+            }
+        } catch (NumberFormatException e) {
+            throw new NumberFormatException();
+        }
+
+        return number;
+    }
+
+    private static int sumNumbers(String[] splitString) {
+        int sum = 0;
+        for( String num :  splitString) {
```

Comment:
자바 컨벤션이 지켜지지 않은 부분이 있네요!
```suggestion
        for (String num : splitString) {
```
아래 링크의 7.5절을 참고하시면 좋을 것 같아요 :)
https://myeonguni.tistory.com/1596

Reply:
반영 완료 했습니다.